internal/pkg/agent/application/monitoring/v1_monitor.go

```diff
@@ -58,7 +58,7 @@ const (
 var (
 	errNoOuputPresent          = errors.New("outputs not part of the config")
 	supportedMetricsComponents = []string{"filebeat", "metricbeat", "apm-server", "auditbeat", "cloudbeat", "fleet-server", "heartbeat", "osquerybeat", "packetbeat", "shipper"}
-	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat"}
+	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat", "pf-host-agent", "pf-elastic-collecor", "pf-elastic-symbolizer"}
```

Comment:
```suggestion
	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat", "pf-host-agent", "pf-elastic-collector", "pf-elastic-symbolizer"}
```

I assume there is supposed to be a t in there. I don't think that we have a reasonable way to test this automatically besides just manually starting the agent.

Reply:
good spot!